include/main.h

```diff
@@ -1 +1,12 @@
+#ifndef __MAIN__H__
```

Comment:
Guidelines for Comments described in Coding Conventions from Professor, Point 3 suggests that "Every file should have a comment at the top describing its contents in a general way"

The description of this file should be added.

Reply:
Suggest your give this file some comments at the top describing its contents